tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -130,40 +173,80 @@ public function userSendsTheFollowingShareInvitationUsingTheGraphApi(string $use
 	 * @param TableNode|null $body
 	 *
 	 * @return void
-	 * @throws Exception
+	 * @throws \GuzzleHttp\Exception\GuzzleException
 	 */
 	public function userCreatesAPublicLinkShareWithSettings(string $user, TableNode  $body):void {
+		$response = $this->userCreatesAPublicLinkShare($user, $body);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" has created the following link share:$/
```

Comment:
```suggestion
	 * @Given /^user "([^"]*)" has created the following link share:$/
```
Isn't it `Given` step?